src/mapping/mod.rs

```diff
@@ -2,6 +2,7 @@ use crate::event::{Event, Value};
 use std::collections::BTreeMap;
 use std::convert::TryFrom;
 
+#[allow(clippy::upper_case_acronyms)] // some of the generated names, like NEWLINE, come from Pest itself https://github.com/pest-parser/pest/issues/49k0
```

Comment:
I wanted to attach this directly to the parser here: https://github.com/timberio/vector/blob/c9a12b31391abb068fa8333bc61cc7867f25eabc/src/mapping/parser/mod.rs#L47

But I couldn't find a way to do it where it would propagate to the generated code. I welcome ideas there.

Reply:
We should be able to delete everything in `mapping` once the Lookup PR #5374  gets merged, which will hopefully not be too far away, so it's probably not worth worrying about this too much.